src/blockchain/mod.rs

```diff
@@ -456,6 +458,64 @@ impl<K: KvStore> KvStoreChain<K> {
             acc_src.nonce += 1;
 
             match &tx.data {
+                TransactionData::CreateToken { token } => {
```

Comment:
It doesn't seem like there's any sort of cost related to creating a custom token? Without some sort of cost to token creation, it would be possible to DoS nodes by creating an infinite number of custom tokens

Reply:
- cost is paid by transaction creator (tx.src). txs are sorted by their fee so if there is a minimum fee, token creator should pay zieshas
 - I thought a general u64 might be better, the token designer can decide its representation on application level (Initial supply of 1 == NFT)
 - initial supply and mintability are both given when creating the token